stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -322,25 +362,26 @@ class PaintedPath:
     def clipping_path(self): ...
     @clipping_path.setter
     def clipping_path(self, new_clipath) -> None: ...
-    def transform_group(self, transform) -> _GeneratorContextManager[Incomplete]: ...
+    @contextmanager
+    def transform_group(self, transform) -> Iterator[Incomplete]: ...
```

Comment:
It yields `self`, and not using `Any` in this method will probably help a lot in type checking. You shouldn't lose autocompletions, type checker errors etc only because you decided to apply a transform :)

```suggestion
    def transform_group(self: Self, transform) -> Iterator[Self]: ...
```